crates/adapters/src/format/json/output.rs

```diff
@@ -62,7 +63,7 @@ impl OutputFormat for JsonOutputFormat {
             )
         })?;
 
-        validate(&config, endpoint_name)?;
+        validate(&config, endpoint_name, schema)?;
```

Comment:
schema may change later too, but I don't know what you can do about it

Reply:
not much, without recompilation.